MarkBind/markbind
# Upgrade vulnerable MarkBind library dependencies

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Other, please explain: Upgrading dependencies


Part of #461, follow-up from #788. 



**What is the rationale for this request?**
`npm audit` reports **22** vulnerabilities in MarkBind's core library (`src/lib/markbind`). We should upgrade vulnerable packages to remove these vulnerabilities.

**What changes did you make? (Give an overview)**
- I upgraded the following vulnerable packages, categoried by their function in `src/lib/markbind`:
	- Building pages
		- `fastmatter`: 2.0.1 → 2.1.0
		- `nunjucks`: 3.0.0 → 3.2.0
		- `markdown-it-table-of-contents`: 0.3.2 → 0.4.3
		- `markdown-it-anchor`: 4.0.0 → 5.0.2
	- Development tools
		- `eslint`: 4.16.0 → 5.15.3
		- `lodash`: 4.17.5 → 4.17.11
- I also upgraded the `eslint` config packages, as there are rule specifications that have changed/deprecated between the `eslint` versions:
	- `eslint-config-airbnb-base`: 12.1.0 → 13.1.0
	- `eslint-plugin-import`: 2.8.0 → 2.16.0
	- `eslint-plugin-lodash`: 2.6.1 → 5.1.0
- I upgraded one package in the main MarkBind package to match the version in `src/lib/markbind`:
	- `markdown-it-anchor`: 5.0.0 → 5.0.2

**Testing instructions:**
1. Ensure that `npm audit` returns 0 vulnerabilities in both the main MarkBind package and `src/lib/markbind`.
2. Ensure that building pages still work eg. with se-edu/se-book.
